atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -204,6 +206,44 @@ public SerializableTransaction(MetricsManager metricsManager,
         return super.getRowsColumnRange(tableRef, rows, columnRangeSelection, batchHint);
     }
 
+    @Override
+    public Map<byte[], Iterator<Map.Entry<Cell, byte[]>>> getRowsColumnRangeIterator(
+            TableReference tableRef,
+            Iterable<byte[]> rows,
+            BatchColumnRangeSelection columnRangeSelection) {
+        Map<byte[], Iterator<Entry<Cell, byte[]>>> ret =
```

Comment:
if this is identical to `getRowsColumnRange` in terms of functionality, should this also be unsupported if serializable?

Reply:
good point - let me revisit it to understand why we had to do this in the first place

Reply:
So it's technically doable, with some caveats, and since no one was using the method with serializable transactions, we chose not to implement it.

The other implementations allow us to track which range you've read as you progress through the BatchingVisitable or Iterator, including marking that you've read the end of the row past the final key if `hasNext()` is false or you visit the last element.

For this method, since we've already combined the results of all the iterators into a single iterator, we can't really reconstruct that data (is this the last key in the row?) without eventually draining the entire iterator anyway.

Reply:
Above comment was about the other method, summary is that we handle it correctly in the new method.